### PR TITLE
Fix various issues based on reviews from the list.

### PR DESCRIPTION
**Pull request by chris-wood**
Importantly, add details regarding incremental deployment, and clarify
label values for non TCP-based variants of TLS.

### CODE REVIEW

File: draft-wood-tls-external-psk-importer.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ differentiate an external PSK into one or more PSKs for use.
 
 Imported keys do not require negotiation for use, as a client and server will not agree upon
 identities if not imported correctly. Thus, importers induce no protocol changes with
-the exception of expanding the set of PSK identities sent on the wire.
+the exception of expanding the set of PSK identities sent on the wire. Endpoints may incrementally
+deploy PSK importer support by offer non-imported keys for TLS versions prior to TLS 1.3.
```
Comment (davidben) [inline draft-wood-tls-external-psk-importer.md]: s/offer/offering/

File: draft-wood-tls-external-psk-importer.md [hunk 2]
```diff
@@ -88,9 +90,10 @@ a tuple of (Base Key, External Identity, KDF). The associated KDF (and hash func
 # Key Import
 
 A key importer takes as input an EPSK with external identity 'external_identity' and base key 'epsk',
-as defined in {{terminology}}, along with an optional label, and transforms it into a set of PSKs and
-imported identities for use in a connection based on supported HashAlgorithms. In particular, for each
-supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity structure as follows:
+as defined in {{terminology}}, along with an optional label, and transforms
+it into a set of PSKs and imported identities for use in a connection based on supported HashAlgorithms.
+In particular, for each supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity
+structure as follows:
```
Comment (davidben) [inline draft-wood-tls-external-psk-importer.md]: (I didn't find any changed text here. Just to confirm, this was just a reflow?)

Comment (chris-wood) [reply draft-wood-tls-external-psk-importer.md]: Yep!

File: draft-wood-tls-external-psk-importer.md (hunk 1 above)
Comment (chris-wood) [reply draft-wood-tls-external-psk-importer.md]: Fixed!

**Pull request merged**